### PR TITLE
Update metrics ClickHouse querying and variables

### PR DESCRIPTION
**Pull request by srikanthccv**

### CODE REVIEW

File: docs/userguide/manage-variables.md [hunk 1]
```diff
@@ -40,21 +40,16 @@ To add a variable to a dashboard, follow the steps below:
...
     ```sql
-    SELECT DISTINCT JSONExtractString(labels, 'job') AS job
-    FROM signoz_metrics.distributed_time_series_v2
-    WHERE metric_name = 'node_cpu_seconds_total'
+    SELECT DISTINCT JSONExtractString(labels, 'host_name') AS host_name
+    FROM signoz_metrics.distributed_time_series_v4_1day
+    WHERE metric_name = 'system_cpu_time'
     ```
 
-    Follwing image shows above case
```
Comment (Calm-Rock) [inline docs/userguide/manage-variables.md:53]: Can we have the image for the updated Query. This will keep things consistent.

**Review by Calm-Rock — DISMISSED** [review]

File: docs/userguide/manage-variables.md (hunk 1 above)
Comment (srikanthccv) [reply docs/userguide/manage-variables.md:53]: Added updated image

**Comment by srikanthccv** [comment]
@Calm-Rock let me know if there are more changes needed.

File: docs/userguide/manage-variables.md [hunk 2]
```diff
@@ -40,20 +40,19 @@ To add a variable to a dashboard, follow the steps below:
...
 
-    Example: You want to query the list of jobs from the metrics database and use them as a variable. Any ClickHouse SQL query that returns a column with desired values is valid. For instance, a query to fetch all jobs that are reporting the CPU metric collection would be
+    Example: You want to query the list of host names from the metrics database and use them as a variable. Any ClickHouse SQL query that returns a column with desired values is valid. For instance, a query to fetch all host names that are reporting the CPU metric collection would be
 
     ```sql
-    SELECT DISTINCT JSONExtractString(labels, 'job') AS job
-    FROM signoz_metrics.distributed_time_series_v2
-    WHERE metric_name = 'node_cpu_seconds_total'
+    SELECT DISTINCT JSONExtractString(labels, 'host_name') AS host_name
+    FROM signoz_metrics.distributed_time_series_v4_1day
```
Comment (prashant-shahi) [inline docs/userguide/manage-variables.md:49]: @srikanthccv Will this include new host names that is not older than a day?

While writing tenant attribute script, I had faced some issue like that.
Had to switch to `distributed_time_series_v4` instead.

```suggestion
    FROM signoz_metrics.distributed_time_series_v4
```

Comment (srikanthccv) [reply docs/userguide/manage-variables.md:49]: It will, your query probably had a condition on unix_milli but this doesn't have the condition.

**Review by ankit01-oss — APPROVED** [review]

**Pull request merged**